dev/cnf/oss_ibm.maven

```diff
@@ -86,10 +86,11 @@ io.smallrye:smallrye-open-api-core:2.1.0-SNAPSHOT-20200917
 io.smallrye:smallrye-open-api-jaxrs:2.1.0-SNAPSHOT-20200917
 net.sf.jtidy:jtidy:9.3.8
 org.apache.aries.blueprint:org.apache.aries.blueprint:1.3.0-ibm-s20170710-0926
-org.apache.cxf:cxf-core:3.3.3.20190529
-org.apache.cxf:cxf-rt-frontend-jaxrs:3.3.3.20190529
-org.apache.cxf:cxf-rt-rs-client:3.3.3.20190529
-org.apache.cxf:cxf-rt-rs-mp-client:3.3.3.20190529
+org.apache.cxf:cxf-core:3.4.1-20201002
+org.apache.cxf:cxf-rt-frontend-jaxrs:3.4.1-20201002
+org.apache.cxf:cxf-rt-rs-client:3.4.1-20201002
+org.apache.cxf:cxf-rt-rs-mp-client:3.4.1-20201002
+org.apache.cxf:cxf-rt-transports-http:3.4.1-20201002
```

Comment:
I would recommend including "ibm" or similar in the snapshot version to make it clear we built it e.g. `org.apache.cxf:cxf-rt-transports-http:3.4.1-ibm20201002`. The wiki for this goes a step further and suggests changing the group id to `com.ibm.ws.org.apache.cxf` ... I think that is overkill.

Reply:
All of these binaries are actual CXF snapshots available (for a limited time) at: 
https://repository.apache.org/content/groups/snapshots/org/apache/cxf/

We're still using the overlay approach (for now) so all IBM changes are part of the Open Liberty repo.  But this PR ends up removing ~30 overlayed files which should make it easier to move off of that approach.